src/odemis/driver/test/technolution_test.py

```diff
@@ -390,13 +390,17 @@ def test_checkMegaFieldExists(self):
         image_received = ASM.checkMegaFieldExists(mega_field_id, "wrong_storage_dir_@#$")
         self.assertFalse(image_received)
 
+    def test_AsmApiException(self):
+        with self.assertRaises(AsmApiException):
+            self.ASM_manager.asmApiGetCall("/fake/function/error", 200, raw_response=True)
```

Comment:
Actually, you could also do the same thing with `asmApiPostCall()`, to check it also generates exceptions properly.